stdlib/asm/crypto/fri/helper.masm

```diff
@@ -67,13 +82,16 @@ export.generate_fri_parameters
         sub.6
         div.2
     end
-    # => [num_fri_layers, remainder_size, lde_size, lde_size, log2(lde_size), domain_gen, 0, ...]
+    # => [num_fri_layers, remainder_size, lde_size, lde_size, log2(lde_size), domain_gen, 0, 0, ...] (12 cycles)
 
-    # Save [num_fri_layers, remainder_size, lde_size, lde_size] in memory
+    # Save `[num_fri_layers, remainder_size, lde_size, lde_size]` in memory
     exec.constants::tmp6 mem_storew
-    swap.5
+    movdn.7
     dropw
     drop
+    drop
+    drop
```

Comment:
These 2 new drops are clearing the stack of the two `0`s loaded together with `lde` size. The idea is that the procedure should leave the stack in the state documented in `Output`.

However, this is not really necessary, because this is called by the `verify` procedure, and the stack is "empty" at that point.

I'm keeping it because it makes refactoring code easier. Let me know if anyone disagrees.

Reply:
Totally agree, it should conform to the expected output stated in the header. Thank you for catching that!